distribution/pom.xml

```diff
@@ -288,6 +288,12 @@
             <groupId>com.hazelcast.jet</groupId>
             <artifactId>hazelcast-jet-files-azure</artifactId>
             <version>${project.version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.apache.commons</groupId>
+                    <artifactId>commons-math3</artifactId>
+                </exclusion>
+            </exclusions>
```

Comment:
This shouldn't be here. 

If the dependencyConvergence reports a version mismatch, you should define the correct version in the root pom in `depencencyManagement` section (which you already have).